Dev

josiah-sanchez opened the pull request.

Commit:
Creation of AssignmentController

Commit:
Updated with addAssignment() method and TestAddAssignment

mizzo23 reviewed (approved):
Keep up the good work Josiah. Also, I think you and I should find a way to hide or remove the application.properties like he mentioned in his video recording and for the next push make sure it doesn't show up.

Commit:
Here is the updated controller with two working JUnit tests. Still
having some issues with JUnitTestDeleteAssignment. Getting an HTTP 400
and not an HTTP 200. Looks like my test assignment isn't saving.
